classes/controllers/class-mondu-orders-controller.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
```

Comment:
Missing 

`if ( ! defined( 'ABSPATH' ) ) {
	exit;
}`

check that is present in other files